Update docs for metric threshold alert filter and group (#1196)
elastic/observability-docs: docs/en/observability/metrics-threshold-alert.asciidoc

In this example, the conditions state that you will receive a critical alert for hosts with a CPU usage of 120% or above and a warning alert if CPU usage is 100% or above. Note that you will receive an alert only if memory usage is 20% or above, as per the second condition.

[role="screenshot"]
image::images/metrics-alert.png[Metric threshold alert]

When you select *Alert me if there's no data*, the rule is triggered if the metrics don't report any data over the
expected time period, or if the rule fails to query {es}.

[discrete]
[[filtering-and-grouping]]
== Filtering and grouping
[role="screenshot"]
image::images/metrics-alert-filters-and-group.png[Metric threshold filter and group fields]

The *Filters* control the scope of the rule. If used, the rule will only evaluate metric data that matches the query in this field. In this example, the rule will only alert on metrics reported from a Cloud region called `us-east`.

> The *Groups alerts by* creates an instance of the alert for every unique value of the `field` added. For example, create a rule per host, or per every mount point of each host. You can also add multiple fields. In this example, the rule will individually track the status of each `host.name` in your infrastructure; if `host.name: host-1` crosses the threshold, but `host-2` and `host-3` do not, you will only receive an alert about `host-1`.

Reviewer: ```suggestion
The *Groups alerts by* creates an instance of the alert for every unique value of the `field` added. For example, you can create a rule per host or every mount point of each host. You can also add multiple fields. In this example, the rule will individually track the status of each `host.name` in your infrastructure. You will only receive an alert about `host-1, if `host.name: host-1` passes the threshold, but `host-2` and `host-3` do not.
```

When you select *Alert me if a group stops reporting data*, the rule is triggered if a group that previously reported metrics does not report them again over the expected time period.

[IMPORTANT]
==============================================

> If you include the same field in both your **Filter** and your **Group by**, you may receive less results than you're expecting. For example, if you have a Filter of `cloud.region: us-east`, then grouping by `cloud.region` will have no effect, because the filter query is only able to match one region.

Reviewer: ```suggestion
If you include the same field in both your **Filter** and your **Group by**, you may receive fewer results than you're expecting. For example, if you filter by `cloud.region: us-east`, then grouping by `cloud.region` will have no effect because the filter query can only match one region.
```
